.ci/bump-cs-version-in-sonar-plugin.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+if [[ -z $1 ]]; then
+  echo "ERROR: version is not set"
+  echo "Usage: bump-cs-version-in-sonar-plugin.sh <version>"
+  exit 1
+fi
+
+VERSION=$1
+SONAR_PLUGIN_FILE=sevntu-checkstyle-sonar-plugin
+
+cd $SONAR_PLUGIN_FILE
```

Comment:
`cd` to  file is nonsense

please remove this variable and use actual values you need.

Reply:
Changes Done